### PR TITLE
allow inline policies through app-interface

### PR DESCRIPTION
**Pull request by maorfr**

### CODE REVIEW

File: utils/terrascript_client.py [hunk 1]
```diff
@@ -486,15 +508,18 @@ def init_common_outputs(self, tf_resources, namespace_info, name):
         output_value = 'Secret'
         tf_resources.append(output(output_name, value=output_value))
 
-    def get_values(self, path):
+    def get_values(self, path, **repalce_args):
```
Comment (jmelis) [inline utils/terrascript_client.py]: typo

File: utils/terrascript_client.py [hunk 2]
```diff
@@ -486,15 +508,18 @@ def init_common_outputs(self, tf_resources, namespace_info, name):
...
         gqlapi = gql.get_api()
         try:
             raw_values = gqlapi.get_resource(path)
+            content = raw_values['content']
         except gql.GqlApiError as e:
             raise FetchResourceError(e.message)
+        for old, new in repalce_args.items():
+            content = content.replace(old, new)
```
Comment (jmelis) [inline utils/terrascript_client.py]: is this supposed to be at the same leve of identation?

File: utils/terrascript_client.py (hunk 1 above)
Comment (maorfr) [reply utils/terrascript_client.py]: thanks!

**Comment by jmelis** [comment]
LGTM

**Pull request merged**